README.md

```diff
@@ -17,6 +17,16 @@ func main() {
 
 	// convert to Slack markdown
 	fmt.Println("Slack: ", githubmarkdownconvertergo.Slack(markdown))
+
+	// optonally githubmarkdownconvertergo.Slack is also accepting 2'nd argument that that can be used to customize converted behavor
+	fmt.Println("Slack: ", githubmarkdownconvertergo.Slack(markdown, githubmarkdownconvertergo.SlackConvertOptions{
+		// Headlines will define if GitHub headlines will be updated to be bold text in slack
+		// there is no headlines as sucks in Slack
+		Headlines: true,
+		// Name of the git repo, used to link pull-requests/issues
+		// repo name to be given in format "<owner>/<name>" (example: eritikass/githubmarkdownconvertergo)
```

Comment:
You can add a note in the readme about how even if the `#25` is an issue, but you use the `/pull/25` url github will automatically redirect to the issue page. I didn't know that until today for example.